firebase-appdistribution/firebase-appdistribution.gradle

```diff
@@ -51,7 +51,7 @@ dependencies {
     runtimeOnly project(':firebase-installations')
 
     testImplementation 'junit:junit:4.13.2'
-    testImplementation "org.robolectric:robolectric:$robolectricVersion"
+    testImplementation "org.robolectric:robolectric:4.8.1"
```

Comment:
I had to update Robolectric to pick up `setAllowInvalidImageData()`, and following the recommendation in Android SDK chat to just update it for our own local tests.